src/main/java/net/mcreator/ui/views/editor/image/tool/tools/FloodFillTool.java

```diff
@@ -44,8 +46,10 @@ public FloodFillTool(Canvas canvas, ColorSelector colorSelector, VersionManager
 		JSlidingSpinner thresholdSlider = new JSlidingSpinner(L10N.t("dialog.image_maker.tools.types.threshold"),
 				threshold, 0, 255, 1);
 		thresholdSlider.addChangeListener(e -> threshold = (int) Math.round(thresholdSlider.getValue()));
+		isGlobal = new JCheckBox(L10N.t("dialog.image_maker.tools.types.floodfill_global"));
 

```

Comment:
I think just calling it "Fill globally" could cause some confusion. In my mind, a global "flood fill" would just fill the entire image with a single color so I think we could add a tooltip explaining what the checkbox does. Something along the line of "Replaces all same-colored pixels with the new color" could probably work.

Reply:
https://www.getpaint.net/doc/latest/PaintBucket.html#:~:text=The%20keyboard%20shortcut%20to%20enable,to%20temporarily%20enable%20Global%20mode.